src/extension.ts

```diff
@@ -281,13 +286,61 @@ export function activate(context: vscode.ExtensionContext): void {
         await logPointsManager.toggleLogpoint(uri);
     });
 
+    initAsyncCommand(context, 'appService.editFile', async (node: IAzureNode<FileTreeItem>) => {
+        await fileEditor.showEditor(node);
+    });
+
+    initEvent(context, 'appService.fileEditor.onDidSaveTextDocument', vscode.workspace.onDidSaveTextDocument, (doc: vscode.TextDocument) => fileEditor.onDidSaveTextDocument(context.globalState, doc));
+    context.subscriptions.push(vscode.commands.registerCommand('diagnostics.LogPoints.OpenScript', openScript));
```

Comment:
Was this a bad merge? I don't understand why you're adding the logpoints command

Reply:
Me neither.  I'm assuming it must have happened on the merge.  Very odd.